pyoxidizer/src/py_packaging/distutils.rs

```diff
@@ -46,70 +46,26 @@ lazy_static! {
 /// modified distutils will survive multiple process invocations, unlike a
 /// monkeypatch. People do weird things in setup.py scripts and we want to
 /// support as many as possible.
-pub fn prepare_hacked_distutils(
-    logger: &slog::Logger,
-    dist: &ParsedPythonDistribution,
-    dest_dir: &Path,
-    extra_python_paths: &[&Path],
-) -> Result<HashMap<String, String>, String> {
-    let extra_sys_path = dest_dir.join("packages");
-
+pub fn prepare_hacked_distutils(logger: &slog::Logger, target: &PythonPaths) {
     warn!(
         logger,
         "installing modified distutils to {}",
-        extra_sys_path.display()
+        target.stdlib.display()
     );
 
-    let orig_distutils_path = dist.stdlib_path.join("distutils");
```

Comment:
This is a separate bug, but it doesnt seem useful to split it off to a separate commit: distutils can not be anywhere - it must be the real one, as setuptools hunts for the real one, even navigating through a venv to find it.